src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/animations';
+
 body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+
+  &__center {
+    height: 5vmin;
+    width: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    position: relative;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+
+    transform-origin: bottom;
+    animation: stopwatch-hend-animation 60s linear infinite;
```

Comment:
The 'stopwatch-hend-animation' used in the animation property is not defined within this file. Ensure that the animation is defined in the imported 'animations.scss' file or define it within this file.